tasks/jsinspect.js

```diff
@@ -92,13 +97,17 @@ module.exports = function(grunt) {
       writableStream.on('finish', function() {
         done(taskSucceeded);
       });
+      writableStream.on('error', function() {
+        throw new Error('Can not write to ' + options.outputPath);
```

Comment:
Are there options that there can be a different error than

```
Fatal error: ENOENT, open 'target/reports/jsinspect/output.xml'
```

If so, we might want to provide some more information here?
Would have to doublecheck, though. But the original error provides at leas `ENOENT`, which gives a hint.